examples/scenario-example-v2.yaml

```diff
@@ -72,6 +69,7 @@ nodes:
   #node_options:
   #  0:
   #    matrix-server: https://transport01.raiden.network
+  #    raiden-client: light-client
```

Comment:
Hm mixing in this "meta" option into the cli options feels a bit "unclean".

Reply:
I know what you mean. Any other ideas? 

I'd also be ok with not having this feature at this point and implementing a "proper" solution later.